src/services/recommendationService.ts

```
@@ -0,0 +1,97 @@
+    } catch (error) {
+      console.error('Error updating recommendation status:', error);
+      throw error;
+    }
+  }
+
+  static async getRecommendationsByTalentIdGroupedByStatus(
+    userId: string,
+    token: string
+  ): Promise<Record<StatusType, RecommendationResponseDTO[]>> {
+    try {
+      const response = await fetch(`${this.BASE_URL}/recommendations/user/${userId}/grouped-by-status`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)